qtaf-core/src/main/java/de/qytera/qtaf/testng/context/QtafTestNGContext.java

```diff
@@ -65,6 +67,11 @@ public abstract class QtafTestNGContext implements IQtafTestContext, AssertionCo
      */
     protected String NO_MESSAGE = "<no-message>";
 
+    /**
+     * Guice injector
+     */
+    protected static final Injector injector = QtafInjector.getInstance();
```

Comment:
Should we make this `private`? Or is there a reason for page objects to have access to this?